src/app/auth/register/register.component.ts

```diff
@@ -53,7 +53,7 @@ export class RegisterComponent {
           this.loading = false;
           Swal.fire({
             title: 'Success!',
-            text: 'Your user id and password created. validate user via email link and login to continue',
+            text: 'Please verify your email by clicking on the link on the email',
```

Comment:
Your user id and password created. Please verify your email by clicking on the link on the email to proceed. If facing any issue, Please contact at ceo@youremployeecheck.com. or whatsapp at +91-8285867446